velox/docs/functions/presto/regexp.rst

```diff
@@ -32,6 +32,22 @@ limited to 20 different expressions per instance and thread of execution.
         SELECT like('abc', '%b%'); -- true
         SELECT like('a_c', '%#_%', '#'); -- true
 
+Not all patterns require compilation of regular expressions. For the patterns without custom escape:
+
+| patterns  | description  | fast search mothod |
```

Comment:
This table might be a nice alternative to current doc, but we'll need to iterate on columns, formatting and user-friendly descriptions. I suggest to do that in a separate PR.